packages/trader/src/Stores/Modules/Trading/Helpers/contract-type.ts

```diff
@@ -696,6 +702,7 @@ export const ContractType = (() => {
 
     return {
         buildContractTypesConfig,
+        getBarrierCategory,
```

Comment:
we will use this method to get barrier_category from contracts_for without setting the value in trade-store because it takes too long for this value to update in trade-store, and as a result we end up with irrelevant one in this case.